contents/handbook/growth/marketing/index.md

```diff
@@ -1,59 +1,97 @@
+- _Everybody codes_ - we know how to write high-quality content for our audience that they’ll actually want to read, and we know the keywords they are using to search for solutions like PostHog. Our more established competitors are ahead of us today, but we know we can overtake them with better content and by targeting low-competition keywords that we’ve identified.
+- _Bias for action_ - we use our team’s extreme autonomy to ship things fast without going through layers of approvals. If we need a week to sign off a social media post, we will probably die. Our paid ads agency doesn’t wait for our approval either if we haven’t responded in 24 hours. 
+ 
+Doesn't sound very exciting on the surface, does it? ('Where are my billboards and flash mobs? Why don't we open a zoo exclusively for hedgehogs? Can we go all in on the metaverse??') 
+ 
+But we think that the opportunity to build a strong brand off the back of these values is genuinely valuable and worthwhile, and leads to far more delightful moments for us and our customers than scrabbling around for magic one-click solutions that invariably don't work. 
+
+## What do we actually do then?
+
+The Marketing team's main focus is acquisition. Our specific objective is to _get as many relevant developers and technical product managers as possible to visit posthog.com_. 
+
+We have identified the following acquisition channels as most important to our strategy. You can find out more details on their relevant pages:
+ 
+- [SEO content](/handbook/growth/marketing/blog). We write high value and genuinely useful content that targets the types of keywords we think are most relevant to our customers. We do 'clean' SEO, ie. no buying backlinks or writing clickbait. 'No clickbait' doesn't mean content has to be boring though! Our aim is to increase our organic reach on Google, basically. 
+- [Brand-building content](/handbook/growth/marketing/blog). We also write content that we think our customers will just find interesting. These may be technical deep dives or articles that give an insight into how PostHog works. We don't expect these to turn into signups, but they help us to build our unique and distinctive brand in the open source community. And frankly, it's just nice to share knowledge with our community about the cool stuff our team is doing. We share this content on 'super node' sites like Hacker News, Product Hunt and Reddit. 
+- [Paid ads](/handbook/growth/marketing/paid). We advertise on Google Search, Twitter, LinkedIn, DuckDuckGo (via Bing) and Reddit. We believe that these sites are most relevant to our audience. While our ultimate objective is to get relevant customers to use PostHog, we don't believe that a customer will click through and then sign up to a $1,500/month contract on the basis of a single ad. However, a paid ad can be the start of a longer term relationship we build with a customer. 
```

Comment:
Our explanation of why we do paid ads sort of doesn't sit well with the rest of this stuff.

I see them as a cheat code to accelerate learning. For example, when we got a couple of pilot paid customers and wanted more, really fast. We perhaps should use them to accelerate learning around which SEO keywords work or to accelerate learning for our newer enterprise product.

Maybe we should frame as "we do paid ads, we use these to accelerate learning. However, we should always focus on scaling through organic growth"

Reply:
Yeah I like that better - will amend